resources/i18n/zh_CN/fdmextruder.def.json.po

```diff
@@ -0,0 +1,203 @@
+# Cura JSON setting files
+# Copyright (C) 2017 Ultimaker
+# This file is distributed under the same license as the Cura package.
+# Ruben Dulek <r.dulek@ultimaker.com>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Cura 2.6\n"
+"Report-Msgid-Bugs-To: r.dulek@ultimaker.com\n"
+"POT-Creation-Date: 2017-06-26 17:50+0800\n"
+"PO-Revision-Date: 2017-06-27 10:23+0800\n"
+"Last-Translator: PCDotFan <pcdotfan@aliyun.com>\n"
+"Language-Team: PCDotFan <pc@edu.ax>\n"
+"Lang-Code: xx\n"
+"Country-Code: XX\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.7.3\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: zh_CN\n"
```

Comment:
Could you change the `Language` to be a human-readable string such as "Simplified Chinese"? Thanks.